lib/device_api/android/adb.rb

```diff
@@ -42,7 +42,7 @@ def self.getprop(serial)
 
         raise ADBCommandError.new(result.stderr) if result.exit != 0
 
-        lines = result.stdout.split("\n")
+        lines = result.stdout.encode('UTF-8', invalid: :replace).split("\n")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.